syntax = "proto3";

package transform.v1.transform;

message TransformRequest {
  string transformationId = 1;
  repeated string data = 2;
}

message TransformResponse {
  bool transformationMissing = 1;
  repeated string data = 2;
}

service Transform {
  rpc TransformData(TransformRequest) returns (TransformResponse);
}
